source/code/scxsystemlib/disk/statisticallogicaldiskinstance.cpp

```diff
@@ -216,7 +216,7 @@ namespace SCXSystemLib
 
         m_timeStamp.AddSample(time(0));
 
-        if (parts.size() == eNumberOfDiskColumns)
+        if (parts.size() >= eNumberOfDiskColumns) // > condition for handling additional fields in kernel 4.18+ and 5.5+
```

Comment:
Use exact number like `parts.size() == eNumberOfDiskColumns || parts.size() == 18 || parts.size() == 20` will be better?
https://www.kernel.org/doc/Documentation/ABI/testing/procfs-diskstats

Reply:
seems `parts.size() >= eNumberOfDiskColumns` is also fine.

Reply:
For now we are only quickly fixing the bug which is preventing the population of the existing disk performance attributes for Kernel version 4.18+.
In long run, we will have additional attributes in the output of SCX_FileSystemStatisticalInformation query based upon kernel version and all.

Reply:
Ensure content shown is appropriate.